src/main/kotlin/xyz/atrius/waystones/commands/WarpstoneCommand.kt

```diff
@@ -65,7 +65,11 @@ object WarpstoneCommand : CommandExecutor {
         player.inventory.addItem(defaultWarpKey(amount))
 
         // Inform Player of given WarpKey
-        sender.sendMessage("&d[Waystones]&r Gave &a$amount&r WarpKey${pluralS(amount)} to &a${player.name}&r".translateColors('&'))
+        sender.sendMessage(
+            "&d[Waystones]&r Gave &a$amount&r WarpKey(s) to &a${player.name}&r"
+                .pluralize(amount)
+                .translateColors('&')
+        )
```

Comment:
I feel like this is spread out a bit too much, maybe compact it like this?
```suggestion
        sender.sendMessage("&d[Waystones]&r Gave &a$amount&r WarpKey(s) to &a${player.name}&r"
        		.pluralize(amount).translateColors('&'))
```